app/views/ips/_table.html.erb

```diff
@@ -1,6 +1,16 @@
 <table class="govuk-table">
   <caption class="govuk-table__caption">
     <%= location.full_address %>
+  <details class="govuk-details govuk-!-margin-bottom-0 govuk-!-margin-top-2">
+    <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      View RADIUS secret key
+    </span>
+  </summary>
+  <div class="govuk-details__text">
+  <%= location.radius_secret_key %>
```

Comment:
nit: indentation
```suggestion
    <%= location.radius_secret_key %>
```

Reply:
Slightly less of a nitpick: It looks like lines 4-13 all need to indent twice (four spaces) more, so that `details` is within ` caption`?